src/bacnet/bacdef.h

```diff
@@ -109,7 +109,7 @@
 #elif (BACNET_PROTOCOL_REVISION == 24)
 /* Addendum 135-2020ca, 135-2020cc, 135-2020bv */
 #define MAX_ASHRAE_OBJECT_TYPE 65
-#define MAX_BACNET_SERVICES_SUPPORTED 47
+#define MAX_BACNET_SERVICES_SUPPORTED 49                /* 2024-05-15 mods4bts This was duplicated elsewhere. BACNET_SERVICES_SUPPORTED_MAX (bacenum.h) cr3817423048721093412*/
```

Comment:
The extra comment should reside in a pull request discussion, but not in the final code.  Please remove the comment.